src/main/java/service/FruitShopService.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface FruitShopService {
+    void startFruitsOperations(List<FruitTransaction> transactions);
```

Comment:
I'd recommend giving more clear naming

Reply:
yesterday i was close to madness because of creating name for this method)) i'll try again 😅